packages/react-components/src/checkbox/src/Checkbox.jsx

```diff
@@ -152,7 +151,7 @@ export function InnerCheckbox(props) {
             {...rest}
             {...wrapperProps}
         >
-            <VisuallyHidden {...inputProps} />
+            <input {...inputProps} />
```

Comment:
Why? I would really prefer if you don't do this.

Reply:
React should throw an exception stating it doesn't know to render the prop `as`.  I am not sure why I don't see this exception in SB preview

Reply:
> Why? I would really prefer if you don't do this and keep VisuallyHidden.
The only way to have an offset on a checkbox since both pseudos are already in use for the checkmark and the border.